### PR TITLE
Generate aliases for use by string parser.

### PR DESCRIPTION
**Pull request by DimitrisJim**
A temporary fix for #4566 so as to not hold back the parser. (This only covers the string parser for now, not `unicodedata`).

### CODE REVIEW

**Comment by DimitrisJim** [comment]
I can open another issue to keep track of the permanent fix for this (and for unicodedata)

File: common/Cargo.toml [hunk 1]
```diff
@@ -4,12 +4,17 @@ version = "0.2.0"
 description = "General python functions and algorithms for use in RustPython"
 authors = ["RustPython Team"]
 edition = "2021"
+build = "build.rs"
```
Comment (youknowone) [inline common/Cargo.toml:7]:
```suggestion
```
This is the default value

**Review by youknowone — COMMENTED** [review]
is this doing the same thing?
https://github.com/RustPython/RustPython/commit/49adb5cd88bc5052f70dd362916940ada572dfb5

**Comment by DimitrisJim** [comment]
> is this doing the same thing? [49adb5c](https://github.com/RustPython/RustPython/commit/49adb5cd88bc5052f70dd362916940ada572dfb5)

does that compile? that's basically what I was starting with but there's some characters that need to be escaped or else `phf` can't really generate the map correctly for them.

**Comment by youknowone** [comment]
Yes, that does compile and working well at least for BACKSPACE. 
`include_str!` obtains the whole file content as a valid str. And I didn't put them in source code but handled only with string operations.

**Comment by DimitrisJim** [comment]
> Yes, that does compile and working well at least for BACKSPACE.

Ah then push it through to here, I have no issue.

> I think something similar like this version can be filed as patch to unicode_names2

I still am not sure for this since they already have the data in [UnicodeData.txt](https://github.com/progval/unicode_names2/blob/master/data/UnicodeData.txt) and they use their own implementation of `phf`. But if you wanna close this PR and go with the patch to `unicode_names2` I'd be fine with that too.

**Comment by youknowone** [comment]
wow, i didn't know their `phf` is their own's.

**Comment by youknowone** [comment]
I tried to add alias, but I found unicode_names2 cannot regenerate master branch's output using the same version of generator 😂 
directly putting your version seems more realistic

**Comment by DimitrisJim** [comment]
Close in favor of #4582

**Comment by youknowone** [comment]
We still need the tests from here, right?

**Pull request closed without merging**